app/src/processing/app/Editor.java

```diff
@@ -1610,8 +1526,6 @@ public int getCurrentTabIndex() {
    */
   public void selectTab(final int index) {
     currentTabIndex = index;
-    undoAction.updateUndoState();
-    redoAction.updateRedoState();
```

Comment:
Removing these calls produce the correct enable/disable of menu actions?
Something to check if this PR is merged.